from PyQt5 import QtCore, QtGui, QtWidgets
from PyQt5.QtWidgets import QMessageBox, QMainWindow, QStyledItemDelegate, QAction
import sys
from database import connect_database
from source import register
class UI_edit_admin(QMainWindow):
    def __init__(self, parent=None):
        super(UI_edit_admin, self).__init__(parent)
        # self.database_config_obj = None
    def setupUi(self):
        self.setObjectName("MainWindow")
        self.resize(545, 455)
        self.centralwidget = QtWidgets.QWidget(self)
        self.setWindowIcon(QtGui.QIcon("../icon/logo.png"))
        self.centralwidget.setObjectName("centralwidget")
        self.label_12 = QtWidgets.QLabel(self.centralwidget)
        self.label_12.setGeometry(QtCore.QRect(170, 0, 181, 41))
        font = QtGui.QFont()
        font.setPointSize(12)
        font.setBold(True)
        font.setWeight(75)
        self.label_12.setFont(font)
        self.label_12.setObjectName("label_12")
        self.label_5 = QtWidgets.QLabel(self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(40, 80, 101, 16))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_5.setFont(font)
        self.label_5.setObjectName("label_5")
        self.label_6 = QtWidgets.QLabel(self.centralwidget)
        self.label_6.setGeometry(QtCore.QRect(40, 130, 101, 16))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_6.setFont(font)
        self.label_6.setObjectName("label_6")
        self.label_7 = QtWidgets.QLabel(self.centralwidget)
        self.label_7.setGeometry(QtCore.QRect(40, 180, 101, 16))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_7.setFont(font)
        self.label_7.setObjectName("label_7")
        self.label_8 = QtWidgets.QLabel(self.centralwidget)
        self.label_8.setGeometry(QtCore.QRect(40, 230, 101, 16))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_8.setFont(font)
        self.label_8.setObjectName("label_8")
        self.label_9 = QtWidgets.QLabel(self.centralwidget)
        self.label_9.setGeometry(QtCore.QRect(40, 280, 101, 16))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_9.setFont(font)
        self.label_9.setObjectName("label_9")
        self.label_10 = QtWidgets.QLabel(self.centralwidget)
        self.label_10.setGeometry(QtCore.QRect(40, 320, 101, 16))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_10.setFont(font)
        self.label_10.setObjectName("label_10")
        self.txt_name_2 = QtWidgets.QLineEdit(self.centralwidget)
        self.txt_name_2.setReadOnly(True)
        self.txt_name_2.setGeometry(QtCore.QRect(200, 70, 301, 31))
        self.txt_name_2.setObjectName("txt_name_2")
        self.txt_name_3 = QtWidgets.QLineEdit(self.centralwidget)
        self.txt_name_3.setGeometry(QtCore.QRect(200, 120, 301, 31))
        self.txt_name_3.setObjectName("txt_name_3")
        self.txt_name_4 = QtWidgets.QLineEdit(self.centralwidget)
        self.txt_name_4.setGeometry(QtCore.QRect(200, 170, 301, 31))
        self.txt_name_4.setObjectName("txt_name_4")
        self.dateEdit = QtWidgets.QDateEdit(self.centralwidget)
        self.dateEdit.setGeometry(QtCore.QRect(200, 220, 181, 31))
        self.dateEdit.setObjectName("dateEdit")
        self.txt_name_5 = QtWidgets.QLineEdit(self.centralwidget)
        self.txt_name_5.setGeometry(QtCore.QRect(200, 270, 301, 31))
        self.txt_name_5.setObjectName("txt_name_5")
        self.txt_name_6 = QtWidgets.QLineEdit(self.centralwidget)
        self.txt_name_6.setGeometry(QtCore.QRect(200, 320, 301, 31))
        self.txt_name_6.setObjectName("txt_name_6")
        self.btn_ok = QtWidgets.QPushButton(self.centralwidget)
        self.btn_ok.setGeometry(QtCore.QRect(100, 370, 89, 31))
        self.btn_ok.setStyleSheet("background-color:rgb(94, 186, 125)")
        self.btn_ok.setObjectName("btn_ok")
        self.btn_canncel = QtWidgets.QPushButton(self.centralwidget)
        self.btn_canncel.setStyleSheet("background-color:rgb(218, 91, 11)")
        self.btn_canncel.setGeometry(QtCore.QRect(310, 370, 89, 31))
        self.btn_canncel.setObjectName("btn_canncel")
        self.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(self)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 545, 25))
        self.menubar.setObjectName("menubar")
        self.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(self)
        self.statusbar.setObjectName("statusbar")
        self.setStatusBar(self.statusbar)
        self.retranslateUi(self)
        # QtCore.QMetaObject.connectSlotsByName(self)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.label_12.setText(_translate("MainWindow", "Edit Profile User"))
        self.label_5.setText(_translate("MainWindow", "ID:"))
        self.label_6.setText(_translate("MainWindow", "User Name:"))
        self.label_7.setText(_translate("MainWindow", "Name:"))
        self.label_8.setText(_translate("MainWindow", "Date of Birth:"))
        self.label_9.setText(_translate("MainWindow", "Email:"))
        self.label_10.setText(_translate("MainWindow", "Address:"))
        self.btn_ok.setText(_translate("MainWindow", "Edit"))
        self.btn_canncel.setText(_translate("MainWindow", "Cancel"))


if __name__ == "__main__":
    app = QtWidgets.QApplication(sys.argv)
    ui = UI_edit_admin()
    ui.setupUi()
    ui.show()
    sys.exit(app.exec_())
